kubeadm/scripts/PrepareNode.ps1

```diff
@@ -32,6 +38,18 @@ function DownloadFile($destination, $source) {
     }
 }
 
+if ($ContainerRuntime -eq "Docker") {
```

Comment:
Kubeadm uses named pipe detection to check for these services: https://github.com/kubernetes/kubernetes/blob/b6b01e17ec8149d53fa9ab5812f9c5947e69a987/cmd/kubeadm/app/util/runtime/runtime_windows.go

May be worth making it consistent? I don't feel strongly about it.

Reply:
Actually I think we should keep it consistent on second thought.

Reply:
Updated